js/calendar.js

```diff
@@ -83,3 +91,153 @@ function refreshCalendar(nextState) {
 			break;
 	}
 }
+
+function loadEvent(event) {
+	let title = "";
```

Comment:
Can declare these inside the `if` statement as they're not used outside of it.

Also, declare the variables as close as we can to where we use them, if possible.

Reply:
Moved declarations